.pylintrc

```diff
@@ -5,5 +5,5 @@ extension-pkg-whitelist=cv2,win32gui,win32process
 generated-members=cv2.*,win32gui.*,win32process.*
 
 [tool.pylint.code_style]
-max-line-length = 160
-disable=broad-except,global-statement,logging-fstring-interpolation,logging-format-interpolation
```

Comment:
I think the `fixme` rule is a good one to keep, (so we have to add exceptions for it the way you had before) since those are generally bad practice 😛